packages/i18n/src/sl.ts

```diff
@@ -0,0 +1,66 @@
+    email:          (issue) => `Neveljaven e-poštni naslov: Prejet ${issue.received}`,
+    emoji:          (issue) => `Neveljaven emoji: Prejet ${issue.received}`,
+    endsWith:       (issue) => `Neveljaven konec: Pričakovan ${issue.expected}, vendar prejet ${issue.received}`,
+    equal:          (issue) => `Neveljavna vrednost: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    excludes:       (issue) => `Neveljavna vsebina: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    finite:         (issue) => `Neveljavno končno število: Prejeto ${issue.received}`,
+    hash:           (issue) => `Neveljaven hash: Prejet ${issue.received}`,
+    hexColor:       (issue) => `Neveljavna šestnajstiška barva: Prejeta ${issue.received}`,
+    hexadecimal:    (issue) => `Neveljavna šestnajstiška številka: Prejeta ${issue.received}`,
+    imei:           (issue) => `Neveljaven IMEI: Prejet ${issue.received}`,
+    includes:       (issue) => `Neveljaven content: Pričakovan ${issue.expected}, vendar prejet ${issue.received}`,
+    integer:        (issue) => `Neveljavno celo število: Prejeto ${issue.received}`,
+    ip:             (issue) => `Neveljaven IP: Prejet ${issue.received}`,
+    ipv4:           (issue) => `Neveljaven IPv4: Prejet ${issue.received}`,
+    ipv6:           (issue) => `Neveljaven IPv6: Prejet ${issue.received}`,
+    isoDate:        (issue) => `Neveljaven datum: Prejet ${issue.received}`,
+    isoDateTime:    (issue) => `Neveljaven datum in čas: Prejet ${issue.received}`,
+    isoTime:        (issue) => `Neveljaven čas: Prejet ${issue.received}`,
+    isoTimeSecond:  (issue) => `Neveljaven iso čas v sekundah: Prejet ${issue.received}`,
+    isoTimestamp:   (issue) => `Neveljaven časovni žig iso: Prejet ${issue.received}`,
+    isoWeek:        (issue) => `Neveljaven format za teden: Prejet ${issue.received}`,
+    length:         (issue) => `Neveljaven dolžina: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    mac:            (issue) => `Neveljaven MAC: Prejet ${issue.received}`,
+    mac48:          (issue) => `Neveljaven 48-bit MAC: Prejet ${issue.received}`,
+    mac64:          (issue) => `Neveljaven 64-bit MAC: Prejet ${issue.received}`,
+    maxBytes:       (issue) => `Neveljaven bajti: Pričakovani ${issue.expected}, vendar prejeti ${issue.received}`,
+    maxLength:      (issue) => `Neveljaven dolžina: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    maxSize:        (issue) => `Neveljaven velikost: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    maxValue:       (issue) => `Neveljaven vrednost: Pričakovana ${issue.expected}, vendar prejeta ${issue.received}`,
+    mimeType:       (issue) => `Neveljaven MIME type: Pričakovan ${issue.expected}, vendar prejet ${issue.received}`,
+    minBytes:       (issue) => `Neveljaven bytes: Pričakovan ${issue.expected}, vendar prejet ${issue.received}`,
```

Comment:
Maybe we could wrote
```suggestion
    minBytes:       (issue) => `Neveljaven bajt: Pričakovan ${issue.expected}, vendar prejet ${issue.received}`,
```